parser_test.go

```diff
@@ -2766,3 +2766,166 @@ func TestParseOutsideDependencies(t *testing.T) {
 		t.Error("Failed to parse api: " + err.Error())
 	}
 }
+
+func TestArrayBody(t *testing.T) {
```

Comment:
This test is useless. We are testing the same thing here:
https://github.com/swaggo/swag/blob/c19280e76fac72a09e9730a4b3c9ae3e144d9b3a/parser_test.go#L2246-L2303